bluesky/callbacks/broker.py

```diff
@@ -27,6 +28,9 @@ def event(self, doc):
             if doc.get('filled', {}).get(field):
                 pass
             else:
+                if self.fs is None:
+                    raise RuntimeError('Either the data must be pre-loaded or'
```

Comment:
I don't like that signature suggests that this parameter is optional (`fs=None`) but that it errors out if it's not provided. Either the parameter should be truly optional, or the signature should make it clear that the parameter is required.